### DISCUSSION TITLE
email issue

### DISCUSSION THREAD

**Discussion by theziya** [frappe/gameplan#272] [Q&A]
while adding people in gameplan  I'm getting error not able to add people to my project it is generating a error message< Please setup default Email Account from Settings > Email Account> i checked settings there is no such ting as email account could u please assist me with this  
![Screenshot from 2024-02-28 13-06-45](https://github.com/frappe/gameplan/assets/108529206/81816e3a-6938-4e5b-91c0-4d42c8c2eaf1)

**Comment by nayaabkhan** [comment]
Facing the same issue
